dashboard/app/controllers/followers_controller.rb

```diff
@@ -64,6 +75,9 @@ def load_section
     # Note that we treat the section as not being found if the section user
     # (i.e., the teacher) does not exist (possibly soft-deleted) or is not a teacher
     unless @section && @section.user&.teacher?
+      if current_user
```

Comment:
NIT: imo `current_user?.increment_section_attempts` is cleaner

Reply:
Is this valid syntax? I would expect an error to the tune of method `current_user?` doesn't exist